src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -32,19 +32,6 @@ PMComplexTest >> testAbsSecure [
 	self assert: (c absSecure equalsTo: 72 sqrt)
 ]
 
-{ #category : #tests }
-PMComplexTest >> testAdaptToCollectionAndSend [
-	"self run: #testAbsSecure"
-
-	"using equalsTo since absSecure returns a slightly different Float"
-
-	| c arr |
-	c := 6 - 6 i.
-	arr := #(0 1 2 3 4 5).
-	self assert: (arr + c at: 1) equals: c.
-	self assert: (arr * c at: 2) equals: c
```

Comment:
Yes, this was a very bad test
Good that you removed it

Reply:
Almost, I split it up, and chose a Conservative route, later seeking support from you and Serge on removing it.